### PR TITLE
Seeds improvements

### PR DESCRIPTION
**Pull request by leio10**

### CODE REVIEW

File: app/queries/decidim/assemblies/prioritized_assemblies.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
```
Comment (leio10) [inline app/queries/decidim/assemblies/prioritized_assemblies.rb]: @deivid-rodriguez This class is redefined to hide child assemblies on `/assemblies` page. Thoughts?

**Comment by leio10** [comment]
Mmm, no, right now I just removed the "db:reset" task from the deployment. We should call it manually when we want.

File: app/queries/decidim/assemblies/prioritized_assemblies.rb (hunk 1 above)
Comment (deivid-rodriguez) [reply app/queries/decidim/assemblies/prioritized_assemblies.rb]: Looks good to me, maybe I would explore if there's the possibility of keeping the original implementation and add the extra filter on top of it.

Comment (leio10) [reply app/queries/decidim/assemblies/prioritized_assemblies.rb]: I've reviewed the original file and we have nothing to keep there: `Decidim::Assembly.order(promoted: :desc)`

Comment (deivid-rodriguez) [reply app/queries/decidim/assemblies/prioritized_assemblies.rb]: The point is to automatically pickup any changes/improvements/fixes if the query changes in core. But if it's not easy to do or desired we can skip it.

**Pull request merged**